tests/unit/api/test_simple.py

```diff
@@ -439,6 +460,15 @@ def test_with_files_with_version_multi_digit(
         if renderer_override is not None:
             assert db_request.override_renderer == renderer_override
 
+
+def _update_context(context, content_type, renderer_override):
+    if renderer_override != "json" or content_type in [
+        simple.MIME_TEXT_HTML,
+        simple.MIME_PYPI_SIMPLE_V1_HTML,
+    ]:
+        return _valid_simple_detail_context(context)
+    return context
```

Comment:
I might be missing something, but I think this helper is currently causing the `test_with_files_quarantined_omitted_from_index` test to be skipped (since it's an instance method, but it's currently indented into the helper.) I'll send a PR to fix it.

Reply:
Opened https://github.com/pypi/warehouse/pull/16777 with the fix.